docs/functions-dev.md

```diff
@@ -42,3 +42,66 @@ To do so set the `--inspect` Node.js option when starting the functions server:
 - On Mac/Linux run `NODE_OPTIONS=--inspect netlify functions:serve`
 
 Then attach any Node.js debugger to the CLI process to debug your functions.
+
+
+## VS Code Debugging
+
+Put the below configuration in your repo's `.vscode/launch.json`.
+
+Then in `VS Code`:
+1. Open `Run and Debug` Sidebar. Make sure "Caught Exceptions" is deactivated.
+2. In the dropdown, select `Fullstack Debugging`
+3. Run the debugger. A chrome window should open. Wait until content is loaded (may take a few seconds for server to be ready).
+
+```jsonc
+{
+  // Use IntelliSense to learn about possible attributes.
```

Comment:
Can we remove the code comments to brevity? 
It seems we don't need the `https://go.microsoft.com/fwlink/?linkid=830387` and the `FULLSTACK NETLIFY DEV DEBUGGING` steps are already described above

Reply:
I think we can remove both in favor of a `to learn more about VSCode launch configurations, see (here)[https://code.visualstudio.com/docs/editor/debugging#_launch-configurations]`

Or something similar. That way we can be brief and provide users the additional information without duplicating it.

Reply:
All right, I will add link towards the official vs code guide and the netlify guide.